strings.txt

```diff
@@ -12094,17 +12094,17 @@ THOUSANDS_SEP
 	DE	.
 	EN	,
 	ES	.
-	FI	 
```

Comment:
Hi Michael,

Sorry, I will revert these changes this evening. I noticed that Atom does remove these. Must be something in Atom's settings. But I checked a few and these seemed to be OK. I should have checked them all ;-)

Greetings,

Martin

Verzonden.

-----Original Message-----
From: mherger <notifications@github.com>
To: Logitech/slimserver <slimserver@noreply.github.com>
Cc: grofweg <aanmartin@gmail.com>, Author <author@noreply.github.com>
Sent: ma, 11 jul. 2016 7:43
Subject: Re: [Logitech/slimserver] Additional Dutch translations (#98)

> @@ -12094,17 +12094,17 @@ THOUSANDS_SEP
>  	DE	.
>  	EN	,
>  	ES	.
> -	FI	 

Hmm... Your editor seems to automatically remove all trailing whitespace. While I don't like it in general as it creates "empty" changes, in this case it's even wrong: the value for FI and PL should be a tab followed by a space. Could you please revert these?

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/Logitech/slimserver/pull/98/files/9185dd83161ad300b6ebc338761127fd38ea8a3f#r70204491